api/gateway.proto

```diff
@@ -365,6 +365,7 @@ message GatewayStatus {
 // Connection stats as monitored by the Gateway Server.
 message GatewayConnectionStats {
   google.protobuf.Timestamp connected_at = 1;
+  google.protobuf.Timestamp disconnected_at = 11;
```

Comment:
We usually write a `// next: 12` at the bottom of the message when we go out-of-order.